### PR TITLE
Last minute tweaks for metadata spec

### PR DESCRIPTION
**Pull request by ptrgags**
Just a couple small tweaks to fix hyperlinks and clarify wording

@sanjeetsuhag could you review?

### CODE REVIEW

File: specification/Metadata/0.0.0/README.md [hunk 1]
```diff
@@ -439,7 +439,7 @@ The JSON encoding is the simplest encoding, designed to be human-readable.
 
 #### Basic Types
 
-For JSON encoding, basic types include integers, floating point numbers, booleans, and strings. These are encoded in the most natural type available in JSON. Numeric types are represented as `number`, booleans as `boolean`, and `string` as `string`.
+For JSON encoding, basic types include integers, floating point numbers, booleans, and strings. These are encoded in whatever JSON type is most. Numeric types are represented as `number`, booleans as `boolean`, and strings as `string`.
```
Comment (sanjeetsuhag) [inline specification/Metadata/0.0.0/README.md:442]:
```suggestion
For JSON encoding, basic types include integers, floating point numbers, booleans, and strings. These are encoded in whatever JSON type is most natural. Numeric types are represented as `number`, booleans as `boolean`, and strings as `string`.
```

**Review by sanjeetsuhag — CHANGES_REQUESTED** [review]

File: specification/Metadata/0.0.0/README.md (hunk 1 above)
Comment (sanjeetsuhag) [reply specification/Metadata/0.0.0/README.md:442]: Also, is natural the right word here? Maybe "appropriate" is better?

Comment (ptrgags) [reply specification/Metadata/0.0.0/README.md]: hm yeah, "appropriate" sounds better here.

**Comment by sanjeetsuhag** [comment]
Fixes look good. Thanks @ptrgags!

**Pull request merged**